packages/radio/tsconfig.build.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["src"],
+  "exclude": [
+    ".storybook",
+    "node_modules",
+    "dist",
+    "**/*.spec.*",
+    "**/*.story.*"
+  ],
+  "compilerOptions": {
+    "declaration": true,
+    "outDir": "dist",
+    "rootDir": "src"
```

Comment:
```suggestion
    "rootDir": "src",
    "strict": true
```

Reply:
Doesn't' setting this in tsconfig.json suffice. If not we really should find a way to stop duplicating ourselves.

Reply:
oh, yeah, I think you're right.  because tsconfig.build.json extends tsconfig.json